packages/data-api/jest.config.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import baseConfig from '../../jest.config-ts.json';
+
+module.exports = async () => ({
+  ...baseConfig,
+  rootDir: '.',
+  setupFilesAfterEnv: ['../../jest.setup.js', './jest.setup.ts'],
+  preset: 'ts-jest',
```

Comment:
Switched to using `ts-jest` for running tests as now we're no longer using `babel` to compile typescript packages